lib/stackup/yaml.rb

```diff
@@ -24,7 +24,8 @@ def dump(*args)
       #   `!Foo blah` as a shortcut for `{ "Fn::Foo" => blah }`
       #
       def load(yaml, filename = nil)
-        tree = ::YAML.parse(yaml, filename)
+        legacy_yaml = Gem::Version.new(RUBY_VERSION) < Gem::Version.new("3.0")
```

Comment:
`legacy_yaml = RUBY_VERSION.split('.')[0].to_i < 3` is a more efficient check, but the above is more readable. I'm easy either way.